internal/adapters/terraform/aws/rds/adapt.go

```diff
@@ -112,6 +112,7 @@ func adaptInstance(resource *terraform.Block, modules terraform.Modules) rds.Ins
 		PublicAccess:              resource.GetAttribute("publicly_accessible").AsBoolValueOrDefault(false, resource),
 		Engine:                    resource.GetAttribute("engine").AsStringValueOrDefault(rds.EngineAurora, resource),
 		IAMAuthEnabled:            resource.GetAttribute("iam_database_authentication_enabled").AsBoolValueOrDefault(false, resource),
+		DeletionProtection:        resource.GetAttribute("deletion_protection").AsBoolValueOrDefault(false, resource),
```

Comment:
Here we are adding the new `DeletionProtection` property to the Terraform adapter - this is pulling the status of Deletion Protection for each DB instance defined in Terraform HCL.